### PR TITLE
doc: Add link to latest api doc version

### PR DESCRIPTION
**Pull request by marikaner**
## Context

This creates a link to the latest version which seems to work, but not completely. I commented out the link from the overview page, but would like to already have this link to create an issue with docusaurus.
I also fixed some spelling errors here and there.

### CODE REVIEW

File: docs/js/api.mdx [hunk 1]
```diff
@@ -17,8 +14,7 @@ image:
 
 import ApiReleaseList from '../../src/sap/sdk-js/ApiReleaseList'
 
-The SAP Cloud SDK for JavaScript and TypeScript consists of multiple libraries to enable you to write side-by-side
-extensions on SAP Cloud Platform. Check `JSDoc`API documentation.
+The SAP Cloud SDK for JavaScript and TypeScript consists of multiple libraries to enable you to write side-by-sideextensions on SAP Cloud Platform. Check `JSDoc`API documentation.
```
Comment (jjtang1985) [inline docs/js/api.mdx:17]:
```suggestion
The SAP Cloud SDK for JavaScript and TypeScript consists of multiple libraries to enable you to write side-by-side extensions on SAP Cloud Platform. Check `JSDoc`API documentation.
```

File: docs/js/redirect-api-reference-js-ts.mdx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+id: api-reference-js-ts
```
Comment (jjtang1985) [inline docs/js/redirect-api-reference-js-ts.mdx:2]: I trust you tested it;)

**Review by jjtang1985 — DISMISSED** [review]
a small fix is needed

File: docs/js/redirect-api-reference-js-ts.mdx (hunk 2 above)
Comment (marikaner) [reply docs/js/redirect-api-reference-js-ts.mdx:2]: Yes, it works 😊

**Review by jjtang1985 — APPROVED** [review]

**Pull request merged**